daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Serializability.scala

```diff
@@ -135,6 +134,15 @@ private[validation] object Serializability {
     template.key.foreach(k => Env(version, world, context, SRKey, k.typ).checkType())
   }
 
+  def checkException(
+      version: LanguageVersion,
+      world: World,
+      tyCon: TTyCon,
+  ): Unit = {
+    val context = ContextTemplate(tyCon.tycon)
+    Env(version, world, context, SRTemplateArg, tyCon).checkType()
```

Comment:
Haskell code uses `SRExceptionArg` here instead of `SRTemplateArg`.

Reply:
Good catch